libs/platform/ebpf_handle.h

```diff
@@ -38,7 +38,7 @@ extern "C"
      *  operation.
      */
     _Must_inspect_result_ ebpf_result_t
-    ebpf_handle_create(ebpf_handle_t* handle, struct _ebpf_base_object* object);
+    ebpf_handle_create(_Out_ ebpf_handle_t* handle, _Inout_ struct _ebpf_base_object* object);
```

Comment:
Maybe update the doxygen as well to be [in,out]? See:
https://github.com/microsoft/ebpf-for-windows/blob/e7302835b1cb443d479cd6b43dfe5838ffe2db84/libs/platform/ebpf_ring_buffer.h#L48
as an example.

Reply:
Right, good point. Updated the doxygen as well.